src/native/libs/System.Security.Cryptography.Native.Browser/pal_crypto_webworker.js

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+const CryptoWebWorkerLib = {
+    $CRYPTOWEBWORKER: {
+        call_digest: function (hash, input_buffer, input_len, output_buffer, output_len) {
+            if (this.can_call_digest() !== 1) {
+                return 0; // Not supported. Caller should have validated this first.
+            }
+
+            var msg = {
+                func: "digest",
+                type: hash,
+                data: Array.from(Module.HEAPU8.subarray (input_buffer, input_buffer + input_len))
+            };
+
+            var response = globalThis.mono_wasm_crypto.channel.send_msg (JSON.stringify (msg));
+            console.info("computed response");
+
+            var digest = JSON.parse (response);
+            console.info("computed digest");
+
+            if (digest.length > output_len) {
+                console.info("call_digest: about to throw!");
+                throw "DIGEST HASH: Digest length exceeds output length: " + digest.length + " > " + output_len;
+            }
+            console.info("validated digest", digest);
+
+            Module.HEAPU8.set (digest, output_buffer);
+            console.info("set digest to output_buffer", digest);
+
+            return 1;
+        },
+        can_call_digest: function () {
+            if (!globalThis.mono_wasm_crypto?.channel || typeof SharedArrayBuffer === "undefined") {
```

Comment:
```suggestion
            if (typeof globalThis.mono_wasm_crypto === "undefined" || typeof globalThis.mono_wasm_crypto.channel === "undefined" || typeof SharedArrayBuffer === "undefined") {) {
```

terser is choking on the new syntax